pkg/providers/tinkerbell/hardware/fmt.go

```diff
@@ -0,0 +1,9 @@
+package hardware
+
+// func formatBMCRef(m Machine) string {
```

Comment:
These are used in a subsequent PR but the linter flags them as deadcode due to deletions hence they're commented out.